src/main/java/hudson/MockExtensionLists.java

```diff
@@ -1,52 +1,65 @@
+/**
+ * This mocks a few different
+ */
+
 package hudson;
 
-import java.util.List;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
-import org.jenkinsci.infra.tools.HyperLocalPluginManager;
 
-import static org.mockito.Mockito.*;
-import org.mockito.stubbing.Answer;
+import org.jenkinsci.infra.tools.HyperLocalPluginManager;
 import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
 import hudson.model.Hudson;
+import hudson.model.listeners.SaveableListener;
 import jenkins.model.Jenkins;
 
-/**
- * A mocked way to get at {@link ExtensionList}s. In {@code hudson} package due to protected access in {@link ExtensionList}.
- */
```

Comment:
I'd retain this comment and improve it rather than replacing it and moving to the top of the file.  It is most common to place Javadoc at the start of the definition of the thing being documented.
```suggestion
/**
 * A mocked way to get at {@link ExtensionList}s. In {@code hudson} package due to protected access in {@link ExtensionList}.
 */
```